.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,31 @@
+name: Build and deploy documentation
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Build
+        run: |
+          npm install
+          # This will be replaced with the actual documentation builder once it's
+          # configured
+          mkdir -p docs-dist/
```

Comment:
As discussed offline, depending on how complex/maintenance-heavy the doc builder becomes, we might want to move it to a separate repo in the future. Then in this action we could just `npm install` that repo. But starting simple with everything in the main repo SGTM 👍

Reply:
Agreed 👍